arm_hw_bridge:
  ros__parameters:
    joint_a:
      min_velocity: -0.05
      max_velocity: 0.05
      min_position: 0.0
      max_position: 0.35
      max_torque: 20.0
      limit_switch_0_present: true
      limit_switch_0_enabled: true
      limit_switch_0_limits_forward: true
      limit_switch_0_active_high: false
      limit_switch_0_used_for_readjustment: true
      limit_switch_0_readjust_position: 0.4
      limit_switch_0_aux_number: 1
      limit_switch_0_aux_pin: 0
      limit_switch_1_present: true
      limit_switch_1_enabled: true
      limit_switch_1_limits_forward: false
      limit_switch_1_active_high: false
      limit_switch_1_used_for_readjustment: true
      limit_switch_1_readjust_position: 0.0
      limit_switch_1_aux_number: 1
      limit_switch_1_aux_pin: 2
    joint_b:
      min_velocity: -1.0
      max_velocity: 1.0
      min_position: -0.9
      max_position: 0.0
      gear_ratio: 1.0
      is_inverted: true
      driver_voltage: 12.0
      motor_max_voltage: 12.0
      quad_present: false
      abs_present: true
      abs_ratio: -1.0
      abs_offset: 0.995
      position_p: 30.0
      velocity_p: 5.0
      calibration_throttle: 0.5
      limit_switch_0_present: true
      limit_switch_0_enabled: true
      limit_switch_0_limits_forward: false
      limit_switch_0_active_high: false
      limit_switch_0_used_for_readjustment: false
    joint_c:
      min_velocity: -0.06
      max_velocity: 0.06
      min_position: -0.125
      max_position: 0.30
      max_torque: 200.0
    joint_de_0:
      min_velocity: -5.0
      max_velocity: 5.0
      min_position: -10000.0
      max_position: 10000.0
      max_torque: 20.0
    joint_de_1:
      min_velocity: -5.0
      max_velocity: 5.0
      min_position: -10000.0
      max_position: 10000.0
      max_torque: 20.0
    gripper:
      gear_ratio: 47.0
      is_inverted: true
      driver_voltage: 10.5
      motor_max_voltage: 12.0
      quad_present: true
      quad_ratio: 1.0
      abs_present: false
      calibration_throttle: 0.5
      position_p: 30.0
      velocity_p: 5.0
      limit_switch_0_present: true
      limit_switch_0_enabled: true
      limit_switch_0_limits_forward: false
      limit_switch_0_active_high: false
      limit_switch_0_used_for_readjustment: true
      limit_switch_0_readjust_position: 0.0
    cam:
      gear_ratio: 1.0
      is_inverted: false
      driver_voltage: 12.0
      motor_max_voltage: 12.0
      quad_present: false
      abs_present: false
      calibration_throttle: 0.5
      limit_switch_0_present: true
      limit_switch_0_enabled: true
      limit_switch_0_limits_forward: true
      limit_switch_0_active_high: false
      limit_switch_0_used_for_readjustment: false
      limit_switch_1_present: true
      limit_switch_1_enabled: true
      limit_switch_1_limits_forward: false
      limit_switch_1_active_high: false
      limit_switch_1_used_for_readjustment: false
